forui/lib/src/theme/color_scheme.dart

```diff
@@ -27,9 +27,6 @@ final class FColorScheme with Diagnosticable {
   /// This is typically used to determine the appearance of native UI elements such as on-screen keyboards.
   final Brightness brightness;
 
-  /// The percentage, between `0` and `1`, used to set a color's lightness to derive a disabled color.
-  final double disabledColorLightness;
```

Comment:
Do you think it's still nice to have this? We could have another for the hover state as well.

Reply:
Ah, to make the opacity configurable?

Reply:
Please add the rename to the `CHANGELOG`